terraform/variables.tf

```diff
@@ -65,7 +76,4 @@ locals {
     values(cloudfoundry_route.education)
   ])
 
-  apply_qts_app_name     = "apply-for-qts-in-england-${var.environment_name}"
-  postgres_database_name = "apply-for-qts-in-england-${var.environment_name}-pg-svc"
```

Comment:
I think we are already passing `environment_name` as  `review` from  review.tfvars.json.  These locals should build the names correctly for review app also. May be it's not needed to pass these as `TF_VARs` from make file into terraform for review app.   Any other reason for removing these 3 local vars ?